addons/mail/tests/discuss/test_message_controller.py

```diff
@@ -385,3 +385,40 @@ def test_mail_cache_control_header(self):
             url=f"/discuss/channel/{self.channel.id}/guest/{self.guest.id}/avatar_128"
         )
         self.assertEqual(res.headers["Cache-Control"], "no-cache")
+
+    def test_chatter_on_archived_record(self):
+        self.authenticate("admin", "admin")
+        archived_partner = self.env["res.partner"].create({"name": "partner", "active": False})
+
+        # 1. posting a message
+        response = self.url_open(
```

Comment:
We could use `make_jsonrpc_request` instead if it's already supported in this version.

Or even better make a tour to test the real flows rather than hard-coding a list RPCs.
We could possibly even just take an existing tour and simply archive its thread.

Reply:
Thanks for the feedback! I didn't know about `make_jsonrpc_request`, i updated the code to use it instead.